notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6321 @@
+{
```

Comment:
<p>I think we can move embedding model id below where we use it so people don't have to go back up. Also you can write why you picked this model and maybe give a link in a markdown cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>++ Done.</p>